twine/commands/register.py

```diff
@@ -16,9 +16,9 @@
 import argparse
 import os.path
 
-from twine import exceptions as exc
 from twine.package import PackageFile
 from twine import settings
+import twine.exceptions
```

Comment:
Again, I'm curious as to why this was needed.

Reply:
It's not required, but I was trying to use the exceptions in a consistent way that makes it clear where they're coming from. There were multiple ways we were importing these exceptions (`from twine import exceptions as exc`, `from twine import exceptions`, etc.) which made it hard to do this refactor, so I was just trying to pick one.

Reply:
So, I don't think you actually need to import `requests.exceptions` just `requests`.

An alternative, could be to do:

```py
# twine/exceptions.py
import requests


class HTTPError(TwineException, requests.exceptions.HTTPError):
    pass
```

And then all we need to do is:

```
from twine import exceptions

try:
    r.raise_for_status()
except exceptions.TwineException as err:
    return _trim_exception(err)
```